index.js

```diff
@@ -13,3 +14,129 @@ closeBtn.addEventListener('click', open);
+  liveLink: 'https://iptysam.github.io/The_Portfolio/',
+  cardImage: 'Images/SnapshootPortfolio1.png',
+  dotImg: 'Images/Counter.png',
+  btags: ['html', 'css', 'javascript', 'github', 'ruby', 'bootstraps'],
+  stags: ['html', 'css', 'javascript'],
+  class: 'container3',
+},
+{
+  id: 'card4',
+  title: 'Multi-post Stories',
+  feature: 'Images/Frame2.png',
+  sdescrip: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  ldescrip: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  liveLink: 'https://iptysam.github.io/The_Portfolio/',
+  cardImage: 'Images/SnapshootPortfolio2.png',
+  btags: ['html', 'css', 'javascript', 'github', 'ruby', 'bootstraps'],
+  stags: ['html', 'css', 'javascript'],
+  class: 'container4',
+},
+];
+
+const cardsEl = document.querySelector('.container');
+allCards.forEach((card) => {
+  cardsEl.innerHTML += `
+  <div class="card1 c-card">
+  <img src="${card.cardImage}" alt="snaphot" class="p-img">
+  <div>
+    <h2>${card.title}</h2>
+    <img src="${card.feature}" alt="counter">  
+    <p>
+    A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="Lang">
+      <li>html</li>
+      <li>css</li>
+      <li>javascript</li>
+    </ul>
+    <button class="button proj">See Project</button>
+  </div> 
+</div>`;
+});
```

Comment:
- [ ] Well done on using javascript to populate your popup windows, however, you still have some little problems in your style to fix, Make sure your design looks like the template one here in this section by using the same button design an layout.

Your design | The template
---|--
![image](https://user-images.githubusercontent.com/65068771/189458436-80190957-be9b-4f9c-bbe8-dcc9d107fbbb.png) | ![image](https://user-images.githubusercontent.com/65068771/189458513-25b7b34a-f4a4-495d-8343-10e696b8af5d.png)